### PR TITLE
Fix typos and add definitions for toxicity detection microservice

### PR DESCRIPTION
**Pull request by tybrs**
## Description

This is just a documentation PR to fix typos and add definitions for toxicity detection microservice.

## Issues
 `n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Just docs changes

### CODE REVIEW

File: comps/guardrails/toxicity_detection/README.md [hunk 1]
```diff
@@ -4,11 +4,13 @@
 
 ## Introduction
 
-Intel also provides toxicity detection model, which is lightweight, runs efficiently on a CPU, and performs well on toxic_chat and jigsaws datasets. More datasets are being fine-tuned. If you're interested, please contact abolfazl.shahbazi@intel.com.
+Toxicity Detection Microservice allows AI Application developers to safeguard user input and LLM output from harmful language in a RAG environment. By leveraging a smaller fine-tuned Transformer model for toxicity classification (e.g. DistilledBERT, RoBERTa, etc.), we maintain a both performant and lightweight guardrails microservice that is readily deployable on both Intel Gaudi and Xeon.
```
Comment (daniel-de-leon-user293) [inline comps/guardrails/toxicity_detection/README.md:7]:
```suggestion
Toxicity Detection Microservice allows AI Application developers to safeguard user input and LLM output from harmful language in a RAG environment. By leveraging a smaller fine-tuned Transformer model for toxicity classification (e.g. DistilledBERT, RoBERTa, etc.), we maintain a performant and lightweight guardrails microservice that is readily deployable on both Intel Gaudi and Xeon.
```

**Review by ashahba — CHANGES_REQUESTED** [review]

File: comps/guardrails/toxicity_detection/README.md [hunk 2]
```diff
@@ -4,11 +4,13 @@
...
 
-## Training Customerizable Toxicity Model on Gaudi2
+Toxicity is defined as rude, disrespectful, or unreasonable language likely to make someone leave a conversation. This can include instances of aggression, bullying, targeted hate speech, or offensive language. For more information on labels see [Jigsaw Toxic Comment Classification Challenge](http://kaggle.com/c/jigsaw-toxic-comment-classification-challenge).
 
-Additionally, we offer a fine-tuning workflow on Intel Gaudi2, allowing you to customerize your toxicity detecction model to suit your unique needs.
+## Future Development
+
+- Add a RoBERTa (125M params) toxicity model fine-tuned on Guaidi with ToxicChat and Jigsaw dataset in an optimized serving framework.
```
Comment (daniel-de-leon-user293) [inline comps/guardrails/toxicity_detection/README.md:13]:
```suggestion
- Add a RoBERTa (125M params) toxicity model fine-tuned on Gaudi with ToxicChat and Jigsaw dataset in an optimized serving framework.
```

Also, do we want to add links to the datasets and model here?

**Review by ashahba — APPROVED** [review]
LGTM!

File: comps/guardrails/toxicity_detection/README.md (hunk 1 above)
Comment (tybrs) [reply comps/guardrails/toxicity_detection/README.md]: I made the language a little clearer here

File: comps/guardrails/toxicity_detection/README.md (hunk 2 above)
Comment (tybrs) [reply comps/guardrails/toxicity_detection/README.md]: Fixed

**Review by mkbhanda — APPROVED** [review]
LGTM

**Review by yinghu5 — APPROVED** [review]

**Pull request merged**